recipes-support/net-snmp/net-snmp_5.9.3.bbappend

```diff
@@ -0,0 +1,7 @@
+EXTRA_OECONF +=  " \
+  --with-security-modules=tsm \
+  --with-transports='DTLSUDP TLSTCP' \
+"
+
+PR="r1"
```

Comment:
`PR` is really needed only if you already released a version out before, otherwise you could also drop for now.